sources/packages/web/src/views/aest/institution/InstitutionDetails.vue

```diff
@@ -0,0 +1,114 @@
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_PROFILE)"
+      ><v-icon size="25" class="mr-2">mdi-city</v-icon>Profile</v-btn
+    >
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_PROGRAMS)"
+    >
+      <v-icon size="25" class="mr-2">mdi-book-open-outline</v-icon
+      >Programs</v-btn
+    >
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_LOCATIONS)"
+      ><v-icon size="25" class="mr-2">mdi-map-marker-outline</v-icon
+      >Locations</v-btn
+    >
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_USERS)"
+      ><v-icon size="25" class="mr-2">mdi-account-group-outline</v-icon
+      >Users</v-btn
+    >
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_DESIGNATION)"
+      ><v-icon size="25" class="mr-2">mdi-certificate-outline</v-icon
+      >Designation</v-btn
+    >
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_RESTRICTIONS)"
+      ><v-icon size="25" class="mr-2">mdi-close-circle-outline</v-icon
+      >Restrictions</v-btn
+    >
+    <v-btn
+      text
+      variant="outlined"
+      @click="showData(AESTRoutesConst.INSTITUTION_NOTES)"
+      ><v-icon size="25" class="mr-2">mdi-clipboard-outline</v-icon>Notes</v-btn
+    >
+    <hr />
+    <router-view v-slot="{ Component }">
+      <keep-alive>
+        <component :is="Component" />
+      </keep-alive>
+    </router-view>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { onMounted, ref } from "vue";
+import { useRouter } from "vue-router";
+import { InstitutionService } from "@/services/InstitutionService";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
+import DesignationStatusBadge from "@/components/generic/DesignationStatusBadge.vue";
+import { BasicInstitutionInfo } from "@/types";
+export default {
+  components: { FullPageContainer, DesignationStatusBadge },
+  props: {
+    institutionId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const router = useRouter();
+    const initialValue = ref({} as BasicInstitutionInfo);
+    const goBack = () => {
+      router.push({
+        name: AESTRoutesConst.SEARCH_INSTITUTIONS,
+      });
+    };
+    const showData = (routeName: symbol) => {
+      router.push({
+        name: routeName,
+        params: { institutionId: props.institutionId },
+      });
+    };
+    onMounted(async () => {
+      initialValue.value = await InstitutionService.shared.getBasicInstitutionInfoById(
+        props.institutionId,
+      );
+      showData(AESTRoutesConst.INSTITUTION_PROFILE);
```

Comment:
Do we need this redirect here?
If the user tries to directly access another route, is this going to prevent him?

Reply:
No it will not, but during the initial load the profile is selected.

Reply:
Are you redirecting to the profile or to the institution detail?